docs/01_nodeos/03_plugins/state_history_plugin/10_how-to-fast-start-without-old-history.md

```diff
@@ -36,7 +36,7 @@ If `nodeos` fails to receive blocks from the network, then try the above using `
 | Either use a firewall to block access to your `http-server-address`, or change it to `localhost:8888` to disable remote access.
 
 [[info]]
-| Whenever you run a filler after this point, use the `--fill-trim` option. Only use `--fpg-create` and `--fill-skip-to` the first time.
+| If you run a database filler after this point, use the `--fill-trim` option. Only use `--fpg-create` and `--fill-skip-to` the first time.
```

Comment:
"...use the \`--fill-trim\` option" -> "...use the \`--fill-trim\` option when restarting."